packages/demo-cards/claim.gts

```diff
@@ -5,25 +5,22 @@ import {
   field,
   StringCard,
   Component,
+  linksTo,
 } from 'https://cardstack.com/base/card-api';
 import { Button, CardContainer, FieldContainer } from '@cardstack/boxel-ui';
 import { tracked } from '@glimmer/tracking';
 // @ts-ignore
+import MetamaskResource from 'metamask-resource';
+// @ts-ignore
 import { enqueueTask, restartableTask } from 'ember-concurrency';
 // @ts-ignore
 import { on } from '@ember/modifier';
 // @ts-ignore
 import { action } from '@ember/object';
 
-declare global {
-  interface Window {
-    ethereum: any;
-  }
-}
-
 class Isolated extends Component<typeof Claim> {
-  @tracked connected: any;
-
+  @tracked chainId = -1;
```

Comment:
I don't think it's correct to keep a separate copy of the chainId here. 

If the goal is to provide a default value when the real `chain` field is undefined, this could be a getter instead:

```js
get chainId() {
  return this.args.model.chain ?? -1;
}
```

Reply:
Resolved https://github.com/cardstack/boxel/pull/372/commits/9c46fcb8bdcb6a85d8c69a6538b66524314fbb40